src/node/certs.h

```diff
@@ -5,5 +5,5 @@
 
 namespace ccf
 {
-  using Certs = kv::Map<Cert, ObjectId>;
+  using Certs = kv::JsonSerialisedMap<Cert, ObjectId>;
```

Comment:
This is what all the changes look like for our governance tables. If they were `kv::Map`, they'd write their Ks and Vs as msgpack, but we want Json so the type of the Map (and only the Map) changes.

We could now go through and remove a bunch of `MSGPACK_DEFINE`s - these types only need to be Json-able, not MsgPack-able. But maybe we'll change our minds, so the `MSGPACK` transformations can stay for now.

Reply:
Let's have a look at the cost of decoding the Cert from JSON on every lookup.